cmd/buildah/commit.go

```diff
@@ -143,7 +144,18 @@ func commitCmd(c *cli.Context) error {
 	}
 	err = builder.Commit(dest, options)
 	if err != nil {
-		return errors.Wrapf(err, "error committing container %q to %q", builder.Container, image)
+		switch nErr := errors.Cause(err).(type) {
+		case errcode.Errors:
+			oErr := nErr[0]
+			for _, e := range nErr[1:] {
+				oErr = errors.Wrapf(e, oErr.Error())
+			}
```

Comment:
Iteratively wrapping each error with a separate call to `errors.Wrapf()` is going to produce duplication in the call stack.  Perhaps converting the `errcode.Errors` into a `github.com/urfave/cli/MultiError` with its `NewMultiError()` function and then wrapping that result by calling `errors.Wrapf()` once would produce something more readable?

Reply:
I found out that `errors.Wrapf`ing `MultiError` makes the error message a little bit confusing so I went with returning just `MultiError` instead.

Reply:
Ah, I was wondering about that.  Makes sense.